20231114_map-based-noise/README.md

```diff
@@ -0,0 +1,32 @@
+LiteBIRD map based simulations of noise
+=======================================
+
+## Summary
+500 realizations of white noise per frequency channel for the LiteBIRD satellite.
+unit = `K_CMB`, nside = 512, coord = Galactic
+
+## Description of simulation
+Maps were generated from white noise covariances provided in the first release of the LiteBIRD end-to-end
+simulations (`LB_e2e_simulations`). The logic that was followed there was simulating all the detectors
+for channels having 48 or less detectors, and simulating roughly 1/3 of the detectors for channels
+with more than 48 detectors (focal plane symmetry was preserved).
+
+## Location
+Folder location: `/global/cfs/cdirs/cmbs4xlb/v1/noise/lb/{channel}`
+where {channel} is one of the 22 channels:
+ - LFT: L1-060, L1-078, L2-050, L2-068, L2-089, L3-068, L3-089, L3-119, L4-078, L4-100, L4-140
+ - MFT: M1-100, M1-140, M1-195, M2-119, M2-166
+ - HFT: H1-195, H1-280, H2-235, H2-337, H3-402
+
+Inside each folder `/global/cfs/cdirs/cmbs4xlb/v1/noise/lb/{channel}` the typical file shape is:
+`{channel}_wn_map_0512_mc_{realization number}.fits`
```

Comment:
are these IQU maps?

Reply:
@toshiyan only open questions are this

Reply:
I am asking Nicolo about the noise simulation description. Can we add him to this repository?

Reply:
Yes, they are I, Q, U